editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -107,9 +124,14 @@ class SpriteFramesEditor : public HSplitContainer {
 	Button *split_sheet_zoom_out = nullptr;
 	Button *split_sheet_zoom_reset = nullptr;
 	Button *split_sheet_zoom_in = nullptr;
+	Button *toggle_settings_button = nullptr;
+	OptionButton *split_sheet_order = nullptr;
 	EditorFileDialog *file_split_sheet = nullptr;
-	HashSet<int> frames_selected;
+	HashMap<int, int> frames_selected; // Key is frame index. Value is selection order.
 	HashSet<int> frames_toggled_by_mouse_hover;
+	Vector<Pair<int, int>> frames_ordered; // First is the index to be ordered by. Second is the actual frame index.
+	int selected_count = 0;
```

Comment:
What is `selected_count` for? Isn't it the same as `frames_selected.size()`?

Reply:
Its for sorting by selected order, if frames are removed and new added the size of the map does not apply

Reply:
I tested the `frames_selected.size()` and it seems to work the same. What exactly could break?

Reply:
It'd take about 68 years if you select one on average every second, just for reference

However I think a different solution will actually work will try it out tomorrow